apps/next/components/header.tsx

```diff
@@ -86,150 +86,6 @@ export default function Header() {
 
     return (
         <>
-            <NewsletterModal isOpen={isOpen} onClose={onClose} />
```

Comment:
Why is all this deleted? It seems as if the header has been deleted. Check if this is right

Reply:
Uhm, I'm not sure, it shouldn't be deleted.